### ISSUE TITLE
Missing "perspective" option?

### ISSUE THREAD

**Issue by aseniah** [ehendrix23/tesla_dashcam#4]
I've installed on a Mac running Mohave via (ffmpeg is already installed):
`pip3 install tesla_dashcam`

When I run `tesla_dashcam -h` or `/usr/local/bin/tesla_dashcam -h` the "perspective" layout option is not listed.
```
  --layout {WIDESCREEN,FULLSCREEN}
                        Layout of the video. Widescreen puts video of all 3
                        cameras next to each other. Fullscreen puts the front
                        camera on top in middle and side cameras below it next
                        to each other (default: WIDESCREEN)
```

However, if I run this command, the "perspective" option does show:
`python3 /usr/local/lib/python3.7/site-packages/tesla_dashcam/tesla_dashcam.py -h`
```
  --layout {WIDESCREEN,FULLSCREEN,PERSPECTIVE}
                        Layout of the created video.
                            PERSPECTIVE: Front camera center top, side cameras next to it in perspective.
                            WIDESCREEN: Output from all 3 cameras are next to each other.
                            FULLSCREEN: Front camera center top, side cameras underneath it.
```

I've tried uninstalling and reinstalling via pip, also neither command seems to indicate the version. Not really a big deal, just a little confusing.

I don't have my dashcam USB drive with me at the moment to test beyond the help view, but I can if you need/want. I'm excited to use this tool, thanks!

**Comment by ehendrix23** [comment]
It looks like the version in /usr/local/bin is a standalone executable and the other one is the one installed from pip.
Not sure why the binary does not list it as if I recall correctly 1st time I did that was with the version that contains perspective.

Note that I currently have a newer release in test. If you wanna try it out: 
https://github.com/ehendrix23/tesla_dashcam/releases/tag/v0.1.8-beta.1

Also has the stand-alone binary there. Still in test hence not yet on pypi for installation with pip. :-)

**Comment by aseniah** [comment]
I tried both versions of 0.1.7 and the 0.1.8 binary last night. The GPU acceleration was quite noticeable! I also like the "mirror" and "rear" options (preference for "rear"). I'm going to close this issue because it's so minor and will likely be resolved once 0.1.8 is released. Thanks for making this awesome tool!
